test-support/helpers/ember-frost-core.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable ember-standard/destructure */
```

Comment:
Why is this added to some file but not others?  I have not yet discovered the pattern requiring it.